src/couch/src/couch_att.erl

```diff
@@ -76,7 +76,7 @@
     %% differs from att_len when encoding /= identity
     disk_len :: non_neg_integer(),
 
-    md5 = <<>> :: binary(),
+    digest = <<>> :: binary(),
```

Comment:
I wonder if there is an online upgrade issue here if we send att records between nodes.

Reply:
Thank you for bringing this up! I was also curious about all the changes in record field names, since it's not obvious at a glance which are shared between nodes.

Reply:
I don't think it matter since records are just tuples on the wire.

Reply:
Ah good point! We're not changing the format or the tuple size here.